to.holepunch.Keet.yml

```diff
@@ -0,0 +1,52 @@
+id: to.holepunch.Keet
+runtime: org.gnome.Platform
+runtime-version: '46'
+sdk: org.gnome.Sdk
+command: /app/bin/usr/bin/keet
+base: org.electronjs.Electron2.BaseApp
+base-version: '23.08'
+finish-args:
+  - --share=ipc
+  - --share=network
+  - --socket=wayland
+  - --socket=x11
+  - --socket=pulseaudio
+  - --device=all
+  - --talk-name=org.kde.StatusNotifierWatcher
+  - --env=XCURSOR_PATH=/run/host/user-share/icons:/run/host/share/icons
+  - --talk-name=org.freedesktop.ScreenSaver
+  - --filesystem=host
```

Comment:
why does a chat program need host access?

Reply:
(I'm not sure this is the right permission, but) the app can send/receive files. Also it needs to install the Pear platform in `.config`

Reply:
it maybe not. it should install stuff in `XDG_CONFIG_HOME`. Also what is that "pear platform"/"pear installer" ? I thought this was chat program

Reply:
it's a messaging app like Discord, but p2p, and it uses our [Pear platform ](https://docs.pears.com/)